proto/kava/pricefeed/v1beta1/pricefeed.proto

```diff
@@ -12,19 +13,25 @@ option (gogoproto.verbose_equal_all) = true;
 message Market {
   option (gogoproto.goproto_stringer) = false;
 
-  string          market_id   = 1 [(gogoproto.customname) = "MarketID"];
-  string          base_asset  = 2;
-  string          quote_asset = 3;
-  repeated string oracles     = 4;
-  bool            active      = 5;
+  string         market_id   = 1 [(gogoproto.customname) = "MarketID"];
+  string         base_asset  = 2;
+  string         quote_asset = 3;
+  repeated bytes oracles     = 4 [
+    (cosmos_proto.scalar) = "cosmos.AddressBytes",
```

Comment:
do you know of any docs on what the `(cosmos_proto.scalar)` does?

Reply:
Doesn't seem to be used yet, maybe later in cosmos v0.45.  Just added to be consistent with swap module

https://github.com/cosmos/cosmos-sdk/issues/9694